org.irssi.irssi.yaml

```diff
@@ -0,0 +1,32 @@
+app-id: org.irssi.irssi
+runtime: org.freedesktop.Platform
+runtime-version: 22.08
+sdk: org.freedesktop.Sdk
+
+command: irssi
+finish-args:
+  - --persist=.irssi
+  - --share=network
+
+modules:
+  - name: irssi
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/irssi/irssi/releases/download/1.4.4/irssi-1.4.4.tar.xz
+        sha256: fefe9ec8c7b1475449945c934a2360ab12693454892be47a6d288c63eb107ead
+        x-checker-data:
+          type: anitya
+          project-id: 1404
+          stable-only: true
+          url-template: https://github.com/irssi/irssi/releases/download/$version/irssi-$version.tar.xz
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 org.irssi.irssi.desktop /app/share/applications/org.irssi.irssi.desktop
+      - install -Dm644 org.irssi.irssi.svg /app/share/icons/hicolor/scalable/apps/org.irssi.irssi.svg
+      - install -Dm644 org.irssi.irssi.svg /app/share/icons/HighContrast/scalable/apps/org.irssi.irssi.svg
+      - install -Dm644 org.irssi.irssi.metainfo.xml /app/share/metainfo/org.irssi.irssi.metainfo.xml
```

Comment:
```suggestion
    post-install:
      - install -Dm644 ../org.irssi.irssi.desktop /app/share/applications/org.irssi.irssi.desktop
      - install -Dm644 ../org.irssi.irssi.svg /app/share/icons/hicolor/scalable/apps/org.irssi.irssi.svg
      - install -Dm644 ../org.irssi.irssi.metainfo.xml /app/share/metainfo/org.irssi.irssi.metainfo.xml
    sources:
      - type: archive
        url: https://github.com/irssi/irssi/releases/download/1.4.4/irssi-1.4.4.tar.xz
        sha256: fefe9ec8c7b1475449945c934a2360ab12693454892be47a6d288c63eb107ead
        x-checker-data:
          type: anitya
          project-id: 1404
          stable-only: true
          url-template: https://github.com/irssi/irssi/releases/download/$version/irssi-$version.tar.xz
```

Reply:
Testing this locally, it fails to build. Why not just have the metadata separately, in its own section?

Reply:
Wait I might just be stupid.

Reply:
Okay, there we go.